WireAPI/Sources/WireAPI/APIs/ConversationsAPI/ConversationsAPIV0.swift

```diff
@@ -96,3 +124,91 @@ private struct PaginatedConversationIDsV0: Decodable, ToAPIModelConvertible {
         )
     }
 }
+
+// MARK: -
+
+struct QualifiedConversationListV0: Decodable, ToAPIModelConvertible {
+    enum CodingKeys: String, CodingKey {
+        case found = "found"
+        case notFound = "not_found"
+        case failed = "failed"
+    }
+
+    let found: [ConversationV0]
+    let notFound: [QualifiedID]
+    let failed: [QualifiedID]
+
+    func toAPIModel() -> ConversationList {
+        ConversationList(
+            found: found.map { $0.toAPIModel() },
+            notFound: notFound,
+            failed: failed
+        )
+    }
+}
+
+// MARK: -
+
+struct ConversationV0: Decodable, ToAPIModelConvertible {
+    enum CodingKeys: String, CodingKey {
```

Comment:
same for those if it's in this file, i would expect to be private otherwise to be in a separate file

Reply:
Please see the answer/discussion in the other conversation: https://github.com/wireapp/wire-ios/pull/1518#discussion_r1628874021